v1/ao/internal/reporter/reporter_grpc.go

```diff
@@ -830,7 +830,11 @@ func (r *grpcReporter) getSettings(ready chan bool) {
 	case errInvalidServiceKey:
 		r.ShutdownNow()
 	case nil:
-		log.Info(method.CallSummary())
+		logger := log.Info
```

Comment:
So it seems like we only print warning response for `getSettings` call for this version? I think it's probably okay if that's the intention. However, it is quite different from what states in https://swicloud.atlassian.net/browse/AO-14723 which does not mention it only applies to `getSettings` call. 

Perhaps we should at least make a mention in either the go or the main jira for this behavior of Go agent? :)

Reply:
Yes, it gets mentioned in the comments in the Jira ticket as `2. only log the warnings fromgetSettings()at default level, log the rest (from postEvents() etc.) at verbose (Golang agent way)`. I've updated the ticket description to make it clearer.

Reply:
Ah got it thanks!

Reply:
Thanks!